analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -115,6 +115,9 @@ class Pub(
         ) = Pub(type, analysisRoot, analyzerConfig, repoConfig)
     }
 
+    // TODO: Maybe enable Gradle functionality only if the plugin is present, and warn if it is not.
+    private val gradleFactory = ALL.getValue("Gradle")
```

Comment:
Line 119 now throws if `Gradle` is not present. Should handling for this be added already now?

Reply:
Done.